core/workspace_comment_svg.js

```diff
@@ -145,12 +145,9 @@ Blockly.WorkspaceCommentSvg.prototype.dispose = function() {
 /**
  * Create and initialize the SVG representation of a workspace comment.
  * May be called more than once.
- *
- * @param {boolean=} opt_noSelect Text inside text area will be selected if false
- *
```

Comment:
I think there was some kind of merge conflict and this accidentally got removed? please add back in.

Reply:
Reverted.